docs/manual/02_concepts/08_avatars/01_animations.md

```diff
@@ -1,5 +1,83 @@
+
+For this reason, world space positional data for avatar bones should be sourced from the raw rig.
+
+The normalized bone space uses quaternions that are relative to the bind pose of the avatar, and is unaffected by world space transformations of the avatar. Quaternion rotations and root transforms written to the normalized rig are retargeted to the raw rig. 
+
+All transforms written to an avatar should be written to the normalized rig for consistency's sake. This guarantees the transformation will have the same effect across all avatars. This consistency is extended to reading a rotation value for a given bone.
+
+## Using Animations
+Avatars can be animated via forward kinematics, e.g. Mixamo animations, or procedurally, with [inverse kinematics](03_IK.md). Player avatars are animated via animation actions that blend animations using [animation graphs](#animation-graphs). You may also place non-player avatars in the studio, which can be [animated with the loop animation component](#creating-animated-avatars-in-the-studio).
+
+### Animation Graphs
+Avatar animations are played in a networked capacity by dispatching the AvatarNetworkAction.setAnimationState action with the name of the preloaded animation pack file, the name of the track from the pack to play, and the avatar entity to apply it to. Optional properties include whether to loop the animation, and its layer. Higher layer values override animations at lower values, where the base locomotion animations are layer 0.
+
+### Creating Animated Avatars In The Studio
+Using the studio, once your avatar is placed in the location, add a loop animation component. If your avatar has its own bundled animation tracks, you may select one in the animation dropdown of the loop animation component. If you'd like to use retargeted Mixamo animations, ensure that the 'Force VRM' is true.
+![Force VRM](./images/force-vrm.jpg)
+
+Next add a URL to an animation pack, then select the animation clip you want to play from the dropdown. You may use either of the two preexisting animation packs in the assets/animations directory of the default project, or [create your own](#creating-animations).
+![Loop Animation](./images/loop-animation.jpg)
+
+## Creating Animations
+This is the recommended workflow for exporting avatar animations, using Blender.
+
+### Export Setup
+When you are ready to export your animation file as GLB, the export will require some minor settings changes to ensure the root motion and animation tracks are exported correctly. 
+
+First, ensure the armature you’re exporting has all transforms applied. An unapplied rotation value will cause root motion axis misalignments. To fix this, press CTRL+A, and select `Rotation` to apply the current rotation.
+
+In the GLB export window, you must set the following values.
+
+If you are using NLA tracks, set the `Animation mode` dropdown to `NLA Tracks`, otherwise set it to `Actions`.
```

Comment:
```suggestion
If you are using NLA tracks, set the **Animation mode** dropdown to **NLA Tracks**, otherwise set it to **Actions**.
```

Fixed bold format for UI elements.